root/application/src/main/java/at/fhv/itb5c/rmi/application/dto/UserConverter.java

```diff
@@ -0,0 +1,40 @@
+package at.fhv.itb5c.rmi.application.dto;
+
+import at.fhv.itb5c.commons.dto.IUser;
+import at.fhv.itb5c.model.entity.User;
+
+public class UserConverter {
+	public static IUser toDTO(User user){
+		UserDTO userdto = new UserDTO();
+		userdto.setId(user.getId());
```

Comment:
`requireNonNull` throws a `NullPointerException` if the param is `null`. I'd rather see a behavior like this:
```
user == null --> return value == null
user != null --> return value != null
```